packages/ui-components/src/components/toolbar.tsx

```diff
@@ -583,6 +583,7 @@ export namespace ToolbarButtonComponent {
     iconLabel?: string;
     tooltip?: string;
     onClick?: () => void;
+    onClickWithEvent?: (e: Event) => void;
```

Comment:
I would support adding it to the existing `onClick` to avoid yet another properties (as we already have `onClick` and `actualOnClick` - let's not add a third one).

But anyway this will need to be revisited when the `Toolbar` widget is used for the cell toolbar.

Reply:
Using `actualOnClick` appears to work. Reverted the `onClickWithEvent` prop. Thanks!